Formula/hdf5-mpi.rb

```diff
@@ -5,6 +5,14 @@ class Hdf5Mpi < Formula
   sha256 "97906268640a6e9ce0cde703d5a71c9ac3092eded729591279bf2e3ca9765f61"
```

Comment:
```suggestion
  sha256 "97906268640a6e9ce0cde703d5a71c9ac3092eded729591279bf2e3ca9765f61"
  license "BSD-3-Clause"
```
This uses the same source as `hdf5`, so we can use the same license.